server/services/conditionService.ts

```diff
@@ -256,10 +256,23 @@ export default class ConditionService {
   /* eslint-disable no-param-reassign */
 
   async getAdditionalAPConditionsForSummaryAndPdf(licence: Licence, user: User): Promise<AdditionalCondition[]> {
+    const conditionDataToExcludeFromSummary = ['nameTypeAndOrAddress']
+
     if (licence.isInPssPeriod && this.isInVariation(licence)) {
       return (await this.licenceApiClient.getParentLicenceOrSelf(licence.id, user))?.additionalLicenceConditions
```

Comment:
this is obv a bit gnarly... keep thinking it'd be good to decorate the returned licence with metadata which would be useful here but its a bit fiddly

Would you need to do the same filtering on this line?

Reply:
yes it does need to be there, fixed.